### ISSUE TITLE
Color contrast 

### ISSUE THREAD

**Issue by lukefretwell** [CityCamp/citycamp.github.io#197]
Feedback:

I can understand someone thinking to use the highest level color contrast (wcag 21.0.1).   However, those two color combinations make it more difficult to some.  Black text against white backgrounds is the most challenging combination for someone with Dyslexia, light sensitivity, Irlen Syndrome, etc.); and, white text against black backgrounds is challenging for people with Astigmatism.
